internal/driver/gomobile/repository.go

```diff
@@ -69,5 +73,20 @@ func (m *mobileFileRepo) Reader(u fyne.URI) (fyne.URIReadCloser, error) {
 }
 
 func (m *mobileFileRepo) Writer(u fyne.URI) (fyne.URIWriteCloser, error) {
+	if u == nil || u.Scheme() != "file" {
+		return nil, repository.ErrOperationNotSupported
+	}
+
+	path := u.Path()
+	_, serr := os.Stat(path)
+	if errors.Is(serr, os.ErrNotExist) {
```

Comment:
This is not going to work. We support Go 1.12 as the minimum version. You'd have to do `os.IsNotExists` instead.